Model/Sinch.php

```diff
@@ -3904,6 +4047,51 @@ private function addCategoryDataMultistoreMerge(
                 ON DUPLICATE KEY UPDATE
                      value = c.Description";
             $this->_doQuery($q);
+
+            $q
```

Comment:
Why create a variable here? (you use it once, then replace its contents). Also the formatting here is unhelpful, why is the variable name on a line of its own, with the equals sign a line below? Why is there a newline at the start of the query (inside the string)?